src/DssSpell.sol

```diff
@@ -15,216 +16,196 @@
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0xBe4De3E151D52668c2C0610C985b4297833239C8,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x58EA3C96a8b81abC01EB78B98deCe2AD1e5fd7fc,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0xBAB4Cd1cB31Cd28f842335973712a6015eB0EcD5,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        (
+            DssVestLike(MCD_VEST_MKR).create(0xB5c86aff90944CFB3184902482799bD5fA3B18dD,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x780f478856ebE01e46d9A432e8776bAAB5A81b5b,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x34364E234b3DD02FF5c8A2ad9ba86bbD3D3D3284,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x46E5DBad3966453Af57e90Ec2f3548a0e98ec979,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x18CaE82909C31b60Fe0A9656D76406345C9cb9FB,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        (
+            DssVestLike(MCD_VEST_MKR).create(0x301dD8eB831ddb93F128C33b9d9DC333210d9B25,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        (
+            DssVestLike(MCD_VEST_MKR).create(0xBFC47D0D7452a25b7d3AA4d7379c69A891bD5d43,       995.00 * 10**18, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        (
+            DssVestLike(MCD_VEST_MKR).create(0xcD16aa978A89Aa26b3121Fc8dd32228d7D0fcF4a,       995.00 * 10**18, SEP_13_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x3189cfe40CF011AAb13aDD8aE7284deD4CD30602,       995.00 * 10**18, JUN_21_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
+        DssVestLike(MCD_VEST_MKR).restrict(
+            DssVestLike(MCD_VEST_MKR).create(0x29b37159C09a65af6a7CFb062998B169879442B6,       995.00 * 10**18, SEP_20_2021,                  4 * 365 days, 365 days,  PE_WALLET)
+        );
```

Comment:
nit
```suggestion
            DssVestLike(MCD_VEST_DAI).create(                                COM_WALLET,   122_700.00 * WAD, SEP_01_2021, JAN_01_2022 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                               DAIF_WALLET,   492_971.00 * WAD, OCT_01_2021, SEP_01_2022 - OCT_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                                GOV_WALLET,   123_333.00 * WAD, SEP_01_2021, OCT_01_2021 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                                GRO_WALLET,   300_050.00 * WAD, SEP_01_2021, NOV_01_2021 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                                MKT_WALLET,   103_134.00 * WAD, SEP_01_2021, NOV_01_2021 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                                ORA_WALLET, 4_196_771.00 * WAD, SEP_01_2021, JUL_01_2022 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                                 PE_WALLET, 4_080_000.00 * WAD, SEP_01_2021, MAY_01_2022 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                               RISK_WALLET, 2_184_000.00 * WAD, SEP_01_2021, SEP_01_2022 - SEP_01_2021,            0, address(0))
        );
        DssVestLike(MCD_VEST_DAI).restrict(
            DssVestLike(MCD_VEST_DAI).create(                                RWF_WALLET,   620_000.00 * WAD, SEP_01_2021, JAN_01_2022 - SEP_01_2021,            0, address(0))
        );

        // Growth MKR whole team vesting
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(GRO_WALLET,                                       803.18 * WAD, JUL_01_2021,                      365 days, 365 days, address(0))
        );

        // Oracles MKR whole team vesting
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(ORA_WALLET,                                     1_051.25 * WAD, JUL_01_2021,                      365 days, 365 days, address(0))
        );

        // PE MKR vestings (per individual)
        (
            DssVestLike(MCD_VEST_MKR).create(0xfDB9F5e045D7326C1da87d0e199a05CDE5378EdD,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0xBe4De3E151D52668c2C0610C985b4297833239C8,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x58EA3C96a8b81abC01EB78B98deCe2AD1e5fd7fc,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0xBAB4Cd1cB31Cd28f842335973712a6015eB0EcD5,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        (
            DssVestLike(MCD_VEST_MKR).create(0xB5c86aff90944CFB3184902482799bD5fA3B18dD,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x780f478856ebE01e46d9A432e8776bAAB5A81b5b,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x34364E234b3DD02FF5c8A2ad9ba86bbD3D3D3284,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x46E5DBad3966453Af57e90Ec2f3548a0e98ec979,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x18CaE82909C31b60Fe0A9656D76406345C9cb9FB,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        (
            DssVestLike(MCD_VEST_MKR).create(0x301dD8eB831ddb93F128C33b9d9DC333210d9B25,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        (
            DssVestLike(MCD_VEST_MKR).create(0xBFC47D0D7452a25b7d3AA4d7379c69A891bD5d43,       995.00 * WAD, MAY_01_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        (
            DssVestLike(MCD_VEST_MKR).create(0xcD16aa978A89Aa26b3121Fc8dd32228d7D0fcF4a,       995.00 * WAD, SEP_13_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x3189cfe40CF011AAb13aDD8aE7284deD4CD30602,       995.00 * WAD, JUN_21_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
        DssVestLike(MCD_VEST_MKR).restrict(
            DssVestLike(MCD_VEST_MKR).create(0x29b37159C09a65af6a7CFb062998B169879442B6,       995.00 * WAD, SEP_20_2021,                  4 * 365 days, 365 days,  PE_WALLET)
        );
```

Reply:
I think gonza mentioned he needed to do 10**18 here because of the decimal multiplication.

Reply:
Nope it will fail for Oracles and Growth MKR distributions. I preferred to keep all the same than having those two different.

Reply:
Sorry I hadn't seen Brian's comment and I was replying to the original one. Yeah what he says is correct.